sdk/csharp/src/Dagger.SDK/Attributes/FieldAttribute.cs

```diff
@@ -0,0 +1,27 @@
+namespace Dagger;
+
+/// <summary>
+/// Marks a property or field as exposed in a Dagger object.
+/// </summary>
+[AttributeUsage(
+    AttributeTargets.Property | AttributeTargets.Field,
+    AllowMultiple = false,
+    Inherited = false
+)]
+public class FieldAttribute : Attribute

```

Comment:
`Field` is not really a thing on dagger because it doesn't really exist in `GraphQL`, everything is a function.

As decided and applied in https://github.com/dagger/dagger/pull/7701, `field` should be  decorated as `func` since these are GQL accesor.

So is it possible to use the `Function` attribute for fields?

Reply:
Ah, I was following ts sdk. Its possible to use the existing `Function` attribute. I will this as a todo to fix for this PR

Reply:
this part is now done